metadata.md

```diff
@@ -1,45 +1,55 @@
+    - Skills recommended: in general, Describers need to have a good handle on metadata practices 
+    - Note that a Gmail email address is required to apply.
+    - Note also that you should be willing to have your real name be associated with the datasets, to follow archival best practices (see [Trust and archival best practices](XXX) for more information).
+  - Credentials, slack invite, Uncrawlable spreadsheet URL, and other details will be provided once your application is approved.
+ - Test that you can get into the ckan instance at https://www.datarefuge.org/ with the credentials provided
+   - Verify that you have write access to the Describers tab in the Uncrawlable spreadsheet
+  - If you need any assistance:
+      - Talk to your DataRescue Guide if you are at an in-person event
+      - Or post questions on Slack in the Describers channel.
 
 - Claiming a bag
-  - In the "Uncrawlable" spreadsheet, locate a bag that has been uploaded and is ready for the CKAN/Metadata step
-  - Such a bag would have the status "Closed" in the cell "Bagger status indicator" 
-  - Claim the bag by entering your slack handle with the status "Open" and today's date in the cell "Describer status indicator" in the Describer section, for instance: 
+  - You will work on datasets that were last checked by Checkers. 
+  - Go to the Uncrawlable spreadsheet, click the Describers tab, and look for a dataset to describe
+    - Available datasets are the ones whose cell "Describer Status Indicator" is empty
+    - If an item is already claimed but its "Date Opened or Closed" cell has turned red, it is also available for you to claim (for more details see the last section of this document)
+  - Claim it by entering your slack handle along with the status "Open" and today's date, for instance: 
   ```
-  @khdelphine open 1/22/2017
+  @khdelphine Open 1/22/2017
   ```
 
 - QA step 
-  - In the "Uncrawlable Content" spreadsheet, click on URL in the "Bag URL" cell in the Bagger section.   
-  - It should start downloading
+  - In the "Uncrawlable Content" spreadsheet, click on URL in the "Bag URL" cell.   
+  - The file should start downloading
   - When it is downloaded, unzip it 
   - Spot check some of the files (make sure they open and look normal, i.e., not garbled)
 
-- Create new record in CKAN 
-  - Click "Add Dataset"
+- Create new record in ckan
+  - Click "Add Dataset" in [ckan](https://www.datarefuge.org/)
   - Start entering metadata following the metadata schema spreadsheet (https://docs.google.com/spreadsheets/d/12JAleU6eF4wgu0hIlQ5efoOUCsZjT-UkRxwAofVZp6c/edit#gid=1879921913)
   - To decide what value to enter in each field:
     - Open JSON file that is in the bag you have downloaded; it contains some of the metadata you need
     - Go to the original location of the item on the federal agency website, to find more more facts about the item such as     description, title of the dataset, etc. (See "Original URL" cell in Unscrawlable spreadsheet.)
```

Comment:
I'm seeing a space between "as" and "description" that looks like  tab is in there